web_src/js/features/repo-graph.ts

```diff
@@ -1,92 +1,64 @@
+  const toggleColorMode = (mode: 'monochrome' | 'colored') => {
+    toggleClass(graphContainer, 'monochrome', mode === 'monochrome');
+    toggleClass(graphContainer, 'colored', mode === 'colored');
+
+    toggleClass(elColorMonochrome, 'active', mode === 'monochrome');
+    toggleClass(elColorColored, 'active', mode === 'colored');
+
     const params = new URLSearchParams(window.location.search);
-    params.set('mode', 'monochrome');
-    const queryString = params.toString();
-    if (queryString) {
-      window.history.replaceState({}, '', `?${queryString}`);
-    } else {
-      window.history.replaceState({}, '', window.location.pathname);
-    }
-    for (const link of document.querySelectorAll('.pagination a')) {
+    params.set('mode', mode);
+    window.history.replaceState(null, '', `?${params.toString()}`);
+    for (const link of document.querySelectorAll('#pagination .pagination a')) {
       const href = link.getAttribute('href');
       if (!href) continue;
       const url = new URL(href, window.location.href);
       const params = url.searchParams;
-      params.set('mode', 'monochrome');
+      params.set('mode', mode);
       url.search = `?${params.toString()}`;
       link.setAttribute('href', url.href);
     }
-  });
+  };
+  elColorMonochrome.addEventListener('click', () => toggleColorMode('monochrome'));
+  elColorColored.addEventListener('click', () => toggleColorMode('colored'));
 
-  document.querySelector('#flow-color-colored')?.addEventListener('click', () => {
-    document.querySelector('#flow-color-colored').classList.add('active');
-    document.querySelector('#flow-color-monochrome')?.classList.remove('active');
-    graphContainer.classList.add('colored');
-    graphContainer.classList.remove('monochrome');
-    for (const link of document.querySelectorAll('.pagination a')) {
-      const href = link.getAttribute('href');
-      if (!href) continue;
-      const url = new URL(href, window.location.href);
-      const params = url.searchParams;
-      params.delete('mode');
-      url.search = `?${params.toString()}`;
-      link.setAttribute('href', url.href);
-    }
-    const params = new URLSearchParams(window.location.search);
-    params.delete('mode');
-    const queryString = params.toString();
-    if (queryString) {
-      window.history.replaceState({}, '', `?${queryString}`);
-    } else {
-      window.history.replaceState({}, '', window.location.pathname);
-    }
-  });
+  const elGraphBody = document.querySelector<HTMLElement>('#git-graph-body');
   const url = new URL(window.location.href);
   const params = url.searchParams;
-  const updateGraph = () => {
+  const loadGitGraph = async () => {
     const queryString = params.toString();
     const ajaxUrl = new URL(url);
     ajaxUrl.searchParams.set('div-only', 'true');
     window.history.replaceState({}, '', queryString ? `?${queryString}` : window.location.pathname);
```

Comment:
```suggestion
    window.history.replaceState(null, '', queryString ? `?${queryString}` : window.location.pathname);
```

Check if those are all cases in the file.

Reply:
They are, committing.

Reply:
There are other `{}` in code.

I believe it is just a nit, no one will be harmed, no bug will occurs, nothing really wrong, right? So why it gets a "block"?

![image](https://github.com/user-attachments/assets/df0b2bdf-810f-44c3-a673-d5ca69cdbc52)

Reply:
I misclicked. Yes we should replace all cases, or introduce a helper function that sets `null, ""` as the first two arguments to `replaceState` and `pushState` automatically.